images/manageiq-base-worker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM manageiq/manageiq-base:latest
+MAINTAINER ManageIQ https://manageiq.org
+
+LABEL name="manageiq-base-worker" \
```

Comment:
Can we drop the "manageiq-"?

Reply:
What is the LABEL for anyway...OpenShift?

Reply:
We probably don't need it, but it's a way to add metadata to an image.  https://docs.docker.com/engine/reference/builder/#label
https://docs.openshift.com/enterprise/3.0/creating_images/metadata.html

Reply:
OpenShift probably interprets them in some way, but our image metadata isn't really in openshift anymore; not since we removed Image Streams anyway. Maybe we can just remove these?

As far as I can tell the only way to see the labels is to `docker inspect` an image and in this case ours shows this:
```json
            "Labels": {
                "build-date": "20170510",
                "description": "ManageIQ is a management and automation platform for virtual, private, and hybrid cloud infrastructures.",
                "io.k8s.description": "ManageIQ is a management and automation platform for virtual, private, and hybrid cloud infrastructures.",
                "io.k8s.display-name": "ManageIQ",
                "io.openshift.tags": "ManageIQ,miq,manageiq",
                "license": "GPLv2",
                "name": "manageiq-base-worker",
                "release": "",
                "summary": "ManageIQ Worker Image",
                "url": "https://manageiq.org/",
                "vendor": "ManageIQ",
                "version": "Master"
           }
```

There's probably some things there we want to change, but it hardly seems necessary to include all of that.